papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -90,9 +90,28 @@ The MDAnalysis library :cite:`Michaud-Agrawal:2011fu` addresses this problem by
 
 The user interface and modular design work equally well in complex scripted workflows, as foundations for other packages :cite:`Seyler:2015fk,Tiberti:2015fk,Somogyi:2016aa`, and for interactive and rapid prototyping work in IPython/Jupyter notebooks, especially together with molecular visualization provided by nglview_ and time series analysis with pandas_. Since the original publication :cite:`Michaud-Agrawal:2011fu`, improvements in speed and data structures make it now possible to work with terabyte-sized trajectories containing up to ~10 million particles. MDAnalysis also comes with specialized analysis classes in the MDAnalysis.analysis module that are unique to MDAnalysis such as the LeafletFinder graph-based algorithm for the analysis of lipid bilayers :cite:`Michaud-Agrawal:2011fu` or the Path Similarity Analysis for the quantitative comparison of macromolecular conformational changes :cite:`Seyler:2015fk`.
 
-MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community. 
+MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community.
 
+Analysis Module
+---------------
 
+In the MDAnalysis.analysis module we provide a large variety of standard analysis algorithms, like RMSD, alignment, native contacts, as well as unique algorithms, like the LeaftleftFinder and Path Similarty Analysis. We have recently started to unify the interface to the different algorithms with an `AnalysisBase` class. Currently PersistenceLength, InterRDF, LinearDensity and Contacts analysis have been ported. If applicable we also strive to make the API's to the algorithms generic. Most other tools hand the user analysis algorithms as black boxes, we want to avoid that and give the users all he needs to adapt an analysis to his/her needs.
```

Comment:
Seriously? That would take a paper in itself. RMSD is pretty easy to define but radius of gyration, alignment those are harder with out context. The path similarity even has it's own paper. I'm happy to cite books for the tasks that give proper context but writing out all formulas for the analysis modules is a bit much.

For the native contacts I can understand this as I go into more detail for that module.